.github/labeler.yml

```
@@ -0,0 +1,24 @@
+# https://github.com/actions/labeler#common-examples
+# Adapted from https://github.com/rapidsai/cuspatial/blob/main/.github/CODEOWNERS
+# Labels culled from https://github.com/rapidsai/cuspatial/labels
+
+Python:
+  - 'python/**'
+  - 'notebooks/**'
+  
+cpp:
+  - 'cpp/**'
+
+CMake:
+  - '**/CMakeLists.txt'
+  - '**/cmake/**'
+
+ci:
```

```suggestion
gpuCI:
```
There's no `ci` label, so this can probably be changed to `gpuCI`.